workflow-templates/shellcheck.yml

```diff
@@ -0,0 +1,65 @@
+# ShellCheck is a linter for your shell scripts:
```

Comment:
What do you think about putting this in a workflow in this repo and then having the template just use that workflow?  This would make it easier to keep things standard and make updates to the upstream actions easier to handle in one place.

Reply:
Discussion moved to https://github.com/openedx/.github/pull/64